src/stories/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,51 @@
+import './styles.scss';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Component for sidebar element.
+ *
+ * @component
+ * @param {string} modifierClasses Class modifiers of the component.
+ * @param {html} sidebarContent Sidebar content of the component.
+ * @param {html} mainContent Main content of the component.
+ * @return {object} (
+ *   <Sidebar modifierClasses={modifierClasses} />
+ *      {sidebarContent}
+ *      {mainContent}
+ *   </Sidebar>
+ * )
+ */
+
+export const Sidebar = ({ modifierClasses, sidebarContent, mainContent}) => {
+    return (
+        <div
+            className={['sidebar', `${modifierClasses}`].join(' ').trim()}
+        >
+            <div>{sidebarContent}</div>
+            <div>{mainContent}</div>
```

Comment:
Add classes to the child divs:
```
<div className="sidebar__aside">{sidebarContent}</div>
<div className="sidebar__main">{sidebarContent}</div>
```

Reply:
I changed things a little bit into left content and right content for more generalization. Hope this is better.